modules/drivers/presto-jdbc/test/metabase/test/data/presto_jdbc.clj

```diff
@@ -24,6 +24,12 @@
 ;; during unit tests don't treat presto as having FK support
 (defmethod driver/supports? [:presto-jdbc :foreign-keys] [_ _] (not config/is-test?))
 
+;; in the past, we had to manually update our Docker image and add a new catalog for every new dataset definition we
+;; added. That's insane. Just use the `test-data` catalog and put everything in that, and use
+;; `db-qualified-table-name` like everyone else.
+(def ^:private test-catalog-name "test_data")
```

Comment:
I was getting Catalog does not exist: database-name when I tried to test presto-jdbc with my custom dataset, so I updated this to make all dataset for presto-jdbc to use the same catalog.

This is what we did for presto too.